azure/deploy-pipeline.yml

```diff
@@ -26,6 +26,7 @@ jobs:
   - job: "Linux"
     pool:
       vmimage: "ubuntu-latest"
+    dependsOn: 'macOS'
```

Comment:
Made each job depend on the last. This will make it take longer but I'm hoping it might make it less likely to timeout